pkg/catalog/diagram.go

```diff
@@ -404,7 +404,11 @@ func (p *Generator) MacroPackages(module *sysl.Module) []string {
 		p.CurrentDir = macroPackageName
 		p.TempDir = macroPackageName // this is for p.Packages()
 		p.Title = macroPackageName
-		p.Links = map[string]string{"Back": "../" + p.OutputFileName}
+		p.Links = map[string]string{
+			"Back":                       "../" + p.OutputFileName,
+			"New bug or feature request": "https://github.service.anz/sysl/api-catalog/issues/new/choose",
```

Comment:
Can we make this a configuration option with a default pointing to https://github.com/anz-bank/sysl-catalog/issues/new rather than hard coded, as these links aren't publicly accessible since `sysl-catalog` is a public tool?